### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by NogsMPLS** [DigitalRiver/react-atlas#10]
We should look into accessibility solutions to integrate with react-atlas. I would love react-atlas to give a soft warning if any given element doesn't have aria- attributes defined, or have aria attributes extracted from other props that have a defaultProp associated with them.

Possibly look into something like https://github.com/rackt/react-a11y to assist with this.

**Comment by stuller** [comment]
https://www.deque.com/axe/ the axe plugin works really well for this too.  Ran it on the docs and only found a couple of issues, most related to contrast.

**Comment by hbowden** [comment]
@stuller So maybe add https://www.npmjs.com/package/axe-core to our CI builds and tests?

**Comment by Darper** [comment]
Maybe if there's a way to throw a warning instead of a failed build. Sometimes we might not have control (or expertise) to make design decisions without outside input. Not sure if the build should fail if contrast isn't high enough.

**Comment by hbowden** [comment]
Personally I would say fail, but if there's too many issues maybe not. Either way it will be nice to add.

**Comment by stuller** [comment]
that would be cool if we could add it in and have it only be a warning - at least until we can get some consensus on the stuff we need to change in the styleguide re: colors, etc.
